tests/pipelines/unidiffuser/test_unidiffuser.py

```diff
@@ -619,6 +673,18 @@ def test_unidiffuser_default_img2text_v1(self):
         expected_text_prefix = "An astronaut"
         assert text[0][: len(expected_text_prefix)] == expected_text_prefix
 
+    @require_torch_2
+    def test_unidiffuser_compile(self, seed=0):
+        inputs = self.get_inputs(torch_device, seed=seed, generate_latents=True)
+        # Delete prompt and image for joint inference.
+        del inputs["prompt"]
+        del inputs["image"]
+        # Can't pickle a Generator object
+        del inputs["generator"]
+        inputs["torch_device"] = torch_device
+        inputs["seed"] = seed
+        run_test_in_subprocess(test_case=self, target_func=_test_unidiffuser_compile, inputs=inputs)
```

Comment:
I think it's okay to skip the compile tests for now. This helps us speed up the SLOW testing suite a bit.

Reply:
I added a `unittest.skip` decorator to the compile test. I think all of the `UniDiffuserSlowTests` got moved to `nightly` in 79a3f39, so maybe this is less important?